src/DocumentDB.ChangeFeedProcessor.UnitTests/PartitionManagement/ChangeFeedDocumentClientExtensions.cs

```diff
@@ -0,0 +1,71 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Azure.Documents.ChangeFeedProcessor.DataAccess;
+using Microsoft.Azure.Documents.ChangeFeedProcessor.UnitTests.Utils;
+using Microsoft.Azure.Documents.Client;
+using Moq;
+
+namespace Microsoft.Azure.Documents.ChangeFeedProcessor.UnitTests.PartitionManagement
+{
+    internal static class ChangeFeedDocumentClientExtensions
+    {
+        public static IChangeFeedDocumentClient SetupQueryResponse(this IChangeFeedDocumentClient client, string pid, string token, string documentLsn, string targetSession, Func<IFeedResponse<Document>, Task<IFeedResponse<Document>>> callback = null)
```

Comment:
please rename pid to somethign like pkRangeId and token to something like continuationToken.

Reply:
also please break line too long.